marklogic-data-hub-spark-connector/src/test/java/com/marklogic/hub/spark/sql/sources/v2/WriteDataTest.java

```diff
@@ -86,9 +86,13 @@ public void testBulkIngestWithoutUriPrefix() throws IOException {
         Map<String, String> params = getHubPropertiesAsMap();
         params.put("batchsize", batchSize);
 
-        if(uriPrefix!=null && uriPrefix.length()!=0) {
+        if (uriPrefix != null && uriPrefix.length() != 0) {
             params.put("uriprefix", uriPrefix);
         }
+        params.put("collections", "fruits,gluefruits");
```

Comment:
See my comment to @anu3990  about an Options class that will simplify writing tests, as opposed to building a Map. We will likely have many tests that populate different sets of options. The Options class will handle that for us.